src/library/index.js

```diff
@@ -17,6 +17,7 @@ export { default as Checkbox, CheckboxGroup } from './Checkbox';
 export { default as Dropdown } from './Dropdown';
 export { default as Flex, FlexItem } from './Flex';
 export { FormField, FormFieldset, FormFieldDivider } from './Form';
+export { default as GridRow, GridItem } from './Grid';
```

Comment:
I believe that `GridRow` should be `Grid`, yes?

Reply:
🤦‍♂️